XenAdmin/TabPages/ManageUpdatesPage.cs

```diff
@@ -215,6 +220,243 @@ private bool FilterIsOn
 
         private void Rebuild()
         {
+            if (byUpdateToolStripMenuItem.Checked)    // By update view
+                RebuildUpdateView();
+            else                                      // By host view
+                RebuildHostView();
+        }
+
+        private class UpdatePageDataGridViewRow : CollapsingPoolHostDataGridViewRow
+        {
+            private class UpdatePageDataGridViewNameCell : DataGridViewExNameCell

```

Comment:
Can't we reuse here the class PatchingWizard_SelectServers.PatchingHostsDataGridViewRow.DataGridViewNameCell to avoid repeating code?  (If so, it will have to move to a more accessible place out of the above nesting).